### ISSUE TITLE
Make Links Work in Graphviz

### ISSUE THREAD

**Issue by jag3773** [unfoldingWord-dev/Door43#60]
The DokuWiki graphviz plugin does not support links, but it needs to!

**Comment by superdav42** [comment]
@jag3773 Are you saying you want a graphiz graph img to be a link?

**Comment by phillip-hopper** [comment]
@superdav42 Yes. This may give you a place to start: http://stackoverflow.com/questions/14155773/label-hyperlink-graphviz

This is the script: https://github.com/Door43/tools/tree/develop/ta

And this is the graph: https://door43.org/en/ta/dependencies

**Comment by jag3773** [comment]
See also https://forum.dokuwiki.org/thread/10575

Thank you,
Jesse Griffin
unfoldingWord Chief Technology Officer

**Comment by superdav42** [comment]
@jag3773 link to a page where we use graphiz?

**Comment by jag3773** [comment]
https://door43.org/en/ta/dependencies

**Comment by superdav42** [comment]
Looks like someone already added support for this in https://github.com/ascendro/dokuwiki-plugin-graphviz PR has been waiting for 3+ years. I suggest we just install his fork.

**Comment by jag3773** [comment]
Cool. Are there any other changes in that fork that we ought to be aware of?

**Comment by superdav42** [comment]
No looks like that's it. It is a little behind the official branch so you'd
loose the Korean translation.

**Comment by phillip-hopper** [comment]
If we fork the official repo, we can merge in the pull request and have the new translations also. If it's been 3 years, they probably aren't maintaining it any more.

**Comment by jag3773** [comment]
Yeah, let's fork it into this org please @phillip-hopper

**Comment by jag3773** [comment]
Where are we at on this @superdav42 or @phillip-hopper ?

**Comment by superdav42** [comment]
@jag3773  Did the forking and merging into my own repo at https://github.com/superdav42/dokuwiki-plugin-graphviz We can install it rom there or fork it into unfoldingWord-dev

**Comment by jag3773** [comment]
@richmahn and/or @phillip-hopper let's get this deployed, whichever way makes the most sense.

**Comment by jag3773** [comment]
Is this in master now?

**Comment by superdav42** [comment]
No, I can update the submodule once the current dev branch is done

**Comment by jag3773** [comment]
Where are we at on this now @superdav42 ?

**Comment by superdav42** [comment]
@jag3773 actually looks like this was already done, I'll test it out.

**Comment by superdav42** [comment]
Seams to work, you can use it like:

``` html
<graphviz>
digraph G {
node1 [URL="http://www.google.com"]
node1 -> node2
}
</graphviz>
```

on any page.

**Comment by jag3773** [comment]
Great!
